.github/workflows/auto-assigning.yml

```diff
@@ -0,0 +1,14 @@
+name: Auto Author Assign
+
+on:
+  pull_request_target:
+    types: [ opened, reopened ]
+
+permissions:
+  pull-requests: write
+
+jobs:
+  assign-author:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: toshimaru/auto-author-assign@v2.1.1
```

Comment:
With the size of this GitHub Action, this should actually be moved into the labeller and rename the labeller to `pull_request_opened.yml`.

This gives the workflow file a meaningful name and compresses what is the same action into 1, easy to identify file.